sdk/go/src/sawtooth_sdk/client/encoder.go

```diff
@@ -0,0 +1,156 @@
+package client
+
+import (
+	"encoding/hex"
+	"fmt"
+	"github.com/golang/protobuf/proto"
+	"sawtooth_sdk/protobuf/batch_pb2"
+	"sawtooth_sdk/protobuf/transaction_pb2"
+	"strings"
+	"time"
+)
+
+// See sdk/go/tests/client_test.go for example usage.
```

Comment:
Technically this line should be part of the next commit. If we rolled back to`Create Go Client SDK`, the file you are referencing would not exist.